convert_state.py

```diff
@@ -1,58 +1,64 @@
+'''
+This program converts between US state names, and abbreviations. 
+For example, converting "MN" to "Minnesota" or converting "California" to "CA".
+'''
+import json
 
 
-
-import json  # import json for json processing
 def main():
     with open('data.json','r') as file:
-        data = file.read()
-        data = json.loads(data)
+        states = file.read()
+        states = json.loads(states)
+    print(states)
 
-    print(data)
-    
     # Create a dictionary of states -> abbreviations, and a dictionary of abbreviations -> states 
-    stateAbbr = data #dictionary of state abbreviations keys and state name values 
-    stateAbbr2 = {}# dictionary of state name keys and state abbreviation values
+    stateAbbr = states    #dictionary of state abbreviations keys and state name values 
+    state = {}     # dictionary of state name keys and state abbreviation values
+    
     # state key is the key in the state dictionary. state value is the value in the dictionary 
-    for statekey, statevalue in stateAbbr.items() :
-        stateAbbr2[statevalue] =  statekey
-
+    for statekey, statevalue in stateAbbr.items():
+        state[statevalue] = statekey
 
-
-    #loop
     while True:
         print('1. Convert state to abbreviation')
-        print("2. Convery abreviasion to state")
-        print(' 3. quit')
-        choice =   input("Enter choice : ")
+        print('2. Convery abbreviation to state')
+        print('3. quit')
+
+        choice = input("Enter choice : ")
 
-        if choice=="1":
+        if choice=='1':
```

Comment:
The spacing or lack of them between choice and the numbers is hard to read (Make it if choice == '1':)

Reply:
Fix the spacing